Automattic/woocommerce-payments
# Display hardware costs in the transaction list and details page

Fixes #1138 

#### Changes proposed in this Pull Request



This PR updates the transactions list to include card reader costs, and also updates the details page to display information about charges for active reader.

Changes:
- add new endpoint to fetch card reader costs
- update the transactions list to include rows with card reader costs. 
![image (1)](https://user-images.githubusercontent.com/82826872/138443728-f16883ab-61d5-4b3f-9f92-be35bb9c0fb4.png)
- update details page to display information about charges for card readers. This screen will be shown when you click the card reader row from transactions list.
<img width="1493" alt="Screenshot 2021-10-22 at 14 06 56" src="https://user-images.githubusercontent.com/82826872/138443937-db0c30cb-480a-4a66-b517-e17048b6802f.png">







#### Testing instructions





* Debit merchant account for active Card Readers used in a month 
* Go to transactions screen and check if the hardware costs are displayed as above
* Click on the hardware costs raw and check if the details are displayed correctly.

-------------------


**Post merge**
